packages/cross_file/lib/src/types/io.dart

```diff
@@ -17,6 +17,15 @@ class XFile extends XFileBase {
   /// [bytes] is ignored; the parameter exists only to match the web version of
   /// the constructor. To construct a dart:io XFile from bytes, use
   /// [XFile.fromData].
+  ///
+  /// [name] is ignored; the parameter exists only to match the web version of
+  /// the constructor. **You should avoid using this parameter because it will
+  /// be likely removed in the future releases.**
```

Comment:
I don't think we should make claims about what's going to be removed; it's much more likely that we would restructure the API in a broader way than just remove parameters.

Reply:
Got it. I'm going the claim "_You should avoid using this parameter because it will be likely removed in the future releases._" from `name`, `length` in constructor and `fromData` factory